### DISCUSSION TITLE
How to nav like in a worktree?

### DISCUSSION THREAD

**Discussion by LintaoAmons** [Bekaboo/dropbar.nvim#168] [Q&A]
Sorry, I have briefly read the README, but I didn't find it~ 

![image](https://github.com/user-attachments/assets/0a0d273f-c31a-4669-ac39-21f51a78ad17)

like I want to use `h/l` to go to the parent dir or the child dir

**Comment by Bekaboo** [comment]
Check https://github.com/Bekaboo/dropbar.nvim?tab=readme-ov-file#menu, you can use `opts.menu.keymaps` option to set keymaps in the drop-down menu. Map `l` to the same function as `<CR>`'s and `h` to `<C-w>q`.

**Reply by LintaoAmons** [reply]
if it's possible to map `h` to navigate to it's parent dir, in this case `lua` folder?

**Reply by Bekaboo** [reply]
> if it's possible to map `h` to navigate to it's parent dir, in this case `lua` folder?

Currently not implemented, you can only close the menus and then pick the `lua` folder from the winbar (I think you can't do this in VSCode either). However, it shouldn't be hard to write a custom function that opens a menu from the previous symbol in the winbar, checkout the [APIs](https://github.com/Bekaboo/dropbar.nvim?tab=readme-ov-file#api) and the [Developer-CLasses](https://github.com/Bekaboo/dropbar.nvim?tab=readme-ov-file#classes) section of the README, you can find all you need to achieve this. Feel free to ask for help if you are confused!

**Reply by LintaoAmons** [reply]
Got! Thanks for the information~ I will try to impl later~ 😈

**Comment by LintaoAmons** [comment]
Another question: Can I disable the change cwd behaviour? I want to keep the cwd as my project dir, and use `dropbar` to nav the files nearby current file. So don't need to change cwd~

**Reply by Bekaboo** [reply]
> Can I disable the change cwd behaviour?

What is the cwd behavior? Dropbar shouldn't change your working directory. Make sure it is not caused by option 'autocwd' or some other plugins. If it keeps troubling you please consider opening a bug report with minimal config & steps to reproduce. Thanks!

**Reply by LintaoAmons** [reply]
Yeah, I checked agian, it was fine~ I thought maybe something else happened last time~  Thanks!
